### DISCUSSION TITLE
HELP for can login for losting auth code and security key BUT I do remember the password of my account

### DISCUSSION THREAD

**Discussion by DreamMakerCS** [community/community#145519] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Dear github manager:

I am jasonleaster who love github very much. Because I changed my job and lost recovery code / authentication code from the old Mac book Pro workstation.

I have stored data and private repository onto github, which are important for me.
**I DO remember my account ID/ email / and the password with it.  sorry for I lost my recovery code.**  

PLEASE， help me to get into my account, I have tried all possible solution services by myself. It didn't work.

my account: jasonleaster

HELP

Yours jasonleaster

**Comment by ebndev** [comment]
Hi @DreamMakerCS,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
